source/includes/stackpath/_edgessl.md

```diff
@@ -31,5 +31,33 @@ Retrieve the EdgeSSL settings for a site in a given [environment](#administratio
 Attributes | &nbsp;
 ------- | -----------
 `id`<br/>*UUID*  | A site's unique identifier. 
-`forceHttps`<br/>*boolean*  | Whether the site redirects all visitors to use HTTPS instead of HTTP. 
-`minTlsVersion`<br/>*string*  | The minimum TLS version clients must have to access the application. 
\ No newline at end of file
+`forceHttps`<br/>*boolean* | Whether the site redirects all visitors to use HTTPS instead of HTTP.
+`minTlsVersion`<br/>*string* | The minimum TLS version clients must have to access the application.
+
+#### Edit EdgeSSL settings
+
+```shell
+curl -X PATCH \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/edgesslsettings/eb3ecdbe-d73b-40e6-a263-166accba75ed"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+    "forceHttps": "true",
+    "minTlsVersion": "V1_2"
+}
+```
+
+<code>PATCH /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/edgesslsettings/:id</a></code>
+
+Edit the EdgeSSL settings for a site in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`forceHttps`<br/>*boolean* | Whether the site redirects all visitors to use HTTPS instead of HTTP.
+`minTlsVersion`<br/>*string* | The minimum TLS version clients must have to access the application.
+
+Only one of these attributes or both can be updated at the same time.
```

Comment:
Nitpick: the sentence can be a bit more concise `One or both of these attributes can be updated at the same time`

Reply:
@aaronalmeida Good suggestion. Will implement it rn. Stand by.